### DISCUSSION TITLE
Iostream error in "BerneseKinematic2Orbit" program

### DISCUSSION THREAD

**Discussion by ParisaShafiei** [groops-devs/groops#126] [Q&A] [answered]
Dear Developers, 

I have found a program called "BerneseKinematic2Orbit" to read the kinematic orbits in AIUB format and convert them to CRF. However, when I used this program to read the AIUB estimated kinematic orbits, I have got this error message: 

**(process  4) basic_ios::clear: iostream error: continue...**

And as a result, the output file is empty. I have attached the configuration file and the results. 
Could yo please tell me why I have gotten this error message?

Thank you in advance for your cooperation and support.

With Kind Regards, 
Parisa 
[AIUB.zip](https://github.com/groops-devs/groops/files/9902194/AIUB.zip)

**Comment by tmayerguerr** [comment, marked as answer]
Dear Parisa,
There was a bug in this program with stations names containing white spaces. Should be fixed now in the main branch.
regards
Torsten

**Reply by ParisaShafiei** [reply]
Dear Torsten, 

Thank you for your hep. I updated the software and now it worked properly. I got the AIUB orbits in CRF. 

With Kind Regards, 
Parisa
